app/main.py

```diff
@@ -1,2 +1,53 @@
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.initial_capacity = 8
+        self.size = 0
+        self.hash_table = [[] for _ in range(self.initial_capacity)]
+        self.threshold = int(self.initial_capacity * 2 / 3)
+
+    def resize(self) -> None:
+        prev_hash_table = self.hash_table
+        self.initial_capacity *= 2
+        self.size = 0
+        self.hash_table = [[] for _ in range(self.initial_capacity)]
+        self.threshold = int(self.initial_capacity * 2 / 3)
+        for item in prev_hash_table:
+            if item:
+                self.__setitem__(item[0], item[1])
```

Comment:
You may use the usual dict assignment interface here, e.g. `self[item[0]] = item[1]`.